src/Components/TodoList.jsx

```diff
@@ -1,11 +1,32 @@
 import { useEffect, useState } from 'react';
-import { toggleComplete, deleteTask } from '../firebase/firestore';
+import {
+  toggleComplete,
+  deleteTask,
+  toggleFavourite,
+} from '../firebase/firestore';
 import { Button } from './Button';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar } from '@fortawesome/free-solid-svg-icons';
+import { faStar as regFaStar } from '@fortawesome/free-regular-svg-icons';
+
 import TodoEdit from './TodoEdit';
 import dateConvert from './utilFunctions/dateConvert';
 
 const TodoList = ({ tasks, getTasksFromFirebase }) => {
   const [todos, setTodos] = useState([]);
+  const fullStar = (
```

Comment:
Please move the icons into a seperate Icons Component end import from there. This will be cleaner and you can use the same icons through out the entire app